index.html

```diff
@@ -14,6 +17,196 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container">
+    <header class="header">
+      <nav class="navigation">
+        <a href="https://wilfredmorgan.com">Home</a>
+        <a href="https://wilfredmorgan.com/#about">About</a>
+        <a href="https://wilfredmorgan.com/#projects">Portfolio</a>
+        <a href="#">Contact</a>
+      </nav>
```

Comment:
I'm glad you could incorporate many of the same styles that you used on your existing portfolio/contact pages. Excellent work